server/neptune/workflows/activities/execute.go

```diff
@@ -17,17 +18,23 @@ type ExecuteCommandResponse struct {
 }
 
 type ExecuteCommandRequest struct {
-	Step    execute.Step
-	Path    string
-	EnvVars map[string]string
+	Step           execute.Step
+	Path           string
+	EnvVars        map[string]string
+	DynamicEnvVars []EnvVar
 }
 
 func (t *executeCommandActivities) ExecuteCommand(ctx context.Context, request ExecuteCommandRequest) (ExecuteCommandResponse, error) {
 	cmd := exec.Command("sh", "-c", request.Step.RunCommand) // #nosec
 	cmd.Dir = request.Path
 
 	finalEnvVars := os.Environ()
-	for key, val := range request.EnvVars {
+
+	requestEnvVars, err := getEnvs(request.EnvVars, request.DynamicEnvVars)
+	if err != nil {
+		return ExecuteCommandResponse{}, errors.Wrap(err, "getting env vars")
```

Comment:
nit: I noticed we don't wrap this error in the terraform activities. Maybe remove this wrap since `getEnvs` error is pretty self-explainatory?

Reply:
Sure that's fine.